controllers/networkplugins_controller.go

```diff
@@ -48,9 +48,9 @@ import (
 const (
 	DefaultNamespace        = "default"
 	MultusImage             = "docker.io/platform9/multus:v3.7.2-pmk-1"
-	WhereaboutsImage        = "docker.io/platform9/whereabouts:v0.4.8"
-	SriovCniImage           = "docker.io/platform9/sriov-cni:v2.6.2-pmk-1"
-	SriovDpImage            = "docker.io/platform9/sriov-network-device-plugin:v3.3.2-pmk-1"
+	WhereaboutsImage        = "docker.io/platform9/whereabouts:v0.4.11"
```

Comment:
No PMK suffix for whereabouts?

Reply:
Yes @cruizen , there is no PMK suffix for whereabouts. We were previously using whereabouts image from the pf9 forked repo only but we branched it out as 0.4.10 and as suggested by mithil, I created branched out 0.4.11 and made changes to it regarding trivy vulnerabilities and created an image

Reply:
Approving for now since we have deviated from https://github.com/k8snetworkplumbingwg/whereabouts/tags : where they have a `release-v0.4` branch and tags for `v0.4.0` to `v0.4.2`

For forked repos, we should use the PMK suffix and the part before that should match the upstream version. We are also probably deviating from the fixes going in `release-v0.4` branch since our tags like `v0.4.10` and `v0.4.11` may not have been rebased to be in sync with the branch.

Reply:
I think it's okay for LTS1 where we want to focus on fixing the security vulnerabilities for now but we should revisit the whereabouts versioning for SMCP GA and LTS2 (I may have missed it if it was part of the discussion on Luigi branching)

Reply:
Yes, we should have had the PMK suffix. Even the previous version of whereabout which we were using in luigi was from forked repo. I continued on the same branch and reduces trivy vulns. But I guess the branches in our PMK whereabouts are a continuation of open source 0.4.2